cmd/operator-ui/templates/blackducks/_form.html

```diff
@@ -71,6 +71,11 @@
       <li value="<%= environ %>"><%= environ %></li>
       <% } %>
     </ul>
+    <input type="hidden" id="blackduck-View.NodeAffinities" name="View.NodeAffinities" value="" />
+    <% } %>
```

Comment:
@yashbhutwala why we need this close button?

Reply:
good catch, fixed.